tests/queries/0_stateless/01600_parts_types_metrics_long.sh

```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env bash
-# Tags: no-s3-storage
```

Comment:
Maybe instead mark it as `no-parallel`?

Reply:
there are still system tables creating parts, right?

Reply:
Yes, but there are retries in the test, `no-parallel` + limited number of retries should help I guess.

Reply:
let's skip it for asan for now as those are the only failures I see for it
if it doesn't help we will see what to do next